packages/mds-metrics-sheet/package.json

```diff
@@ -19,7 +19,9 @@
     "@mds-core/mds-logger": "0.1.9",
     "@mds-core/mds-providers": "0.1.9",
     "@mds-core/mds-types": "0.1.6",
+    "@types/lodash": "4.14.138",
```

Comment:
This should be removed and re-installed as a dev dependency in the root package.json `yarn add -D -W @types/lodash`